source/ui/actions3/AMActionRunnerCurrentViewBase.cpp

```diff
@@ -176,6 +178,12 @@ void AMActionRunnerCurrentViewBase::onCancelButtonClicked()
 	}
 }
 
+void AMActionRunnerCurrentViewBase::onActionRunnerPausableChanged(bool pausable)
+{
+	pauseButton_->setEnabled(pausable);
+	pauseButton_->repaint(); // force repaint;
```

Comment:
If memory serves me correctly, I think you want to call `update()` rather than repaint.  Conversely, why do you need to force it to repaint anyway?  Usually widgets are pretty responsive to this kind of change.